src/solvers/sweep_plans/sweep_plans.jl

```diff
@@ -71,12 +73,13 @@ function forward_sweep(
   regions = collect(
     flatten(map(i -> forward_region(edges, i; region_kwargs, kwargs...), eachindex(edges)))
   )
-
   if reverse_step
     reverse_regions = collect(
       flatten(
         map(
-          i -> reverse_region(edges, i; region_kwargs=reverse_kwargs, kwargs...),
+          i -> reverse_region(
+            edges, i; reverse_edge=reverse_edges, region_kwargs=reverse_kwargs, kwargs...
+          ),
           eachindex(edges),
         ),
```

Comment:
I would prefer using do-block syntax here (https://docs.julialang.org/en/v1/manual/functions/#Do-Block-Syntax-for-Function-Arguments-1) since the function being passed to `map` is pretty long.